pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go

```diff
@@ -44,6 +44,7 @@ const (
 	klusterletApplied            = "Applied"
 	klusterletReadyToApply       = "ReadyToApply"
 	hubConnectionDegraded        = "HubConnectionDegraded"
+	hubKubeConfigSecretMissing   = "HubKubeConfigSecretMissing"
```

Comment:
should this be a const somewhere that statuscontroller can also use?

Reply:
I will do it as a followup.  there are several const variables/strings need to be used as a global one.
not sure which package is the proper place, maybe need to create a new package.